patches/server/0920-Fix-MC-210802.patch

```diff
@@ -0,0 +1,36 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: BillyGalbreath <Blake.Galbreath@Gmail.com>
+Date: Sat, 15 Jan 2022 06:23:04 -0600
+Subject: [PATCH] Fix MC-210802
+
+
+diff --git a/src/main/java/net/minecraft/server/level/ChunkMap.java b/src/main/java/net/minecraft/server/level/ChunkMap.java
+index 5461bac37b6dc0575cccd6656b48b2ef18cfaa04..ec67d9044a50fdc9da8d6dd2d9eae2688452dc34 100644
+--- a/src/main/java/net/minecraft/server/level/ChunkMap.java
++++ b/src/main/java/net/minecraft/server/level/ChunkMap.java
+@@ -983,7 +983,7 @@ public class ChunkMap extends ChunkStorage implements ChunkHolder.PlayerProvider
+         return this.anyPlayerCloseEnoughForSpawning(this.getUpdatingChunkIfPresent(chunkcoordintpair.toLong()), chunkcoordintpair, reducedRange);
+     }
+ 
+-    final boolean anyPlayerCloseEnoughForSpawning(ChunkHolder playerchunk, ChunkPos chunkcoordintpair, boolean reducedRange) {
++    public final boolean anyPlayerCloseEnoughForSpawning(ChunkHolder playerchunk, ChunkPos chunkcoordintpair, boolean reducedRange) { // Paper - package -> public
```

Comment:
AT

Reply:
Working on it, but I'm having issues :3

This should work, but it's not (no error on apply, either). ATs confuse me lol

```
# Fix MC-210802
public net.minecraft.server.level.ChunkMap anyPlayerCloseEnoughForSpawning(Lnet/minecraft/server/level/ChunkHolder;Lnet/minecraft/world/level/ChunkPos;Z)Z
```

Reply:
> It's a paper added method
> So the at won't find it since it's ran before that
> In that case, just change the existing patch

Got it \o/